### DISCUSSION TITLE
怎么才能让gpt回复的代码是处于黑框中？现在回复的代码老是文本形式，看起来难受。

### DISCUSSION THREAD

**Discussion by liangyurong** [timqian/openprompt.co#67] [General]
我自己创建了一个重构代码的prompt，但是回复的代码一直都是文本形式
![image](https://github.com/timqian/openprompt.co/assets/28131328/fa9fa439-1aff-4d17-a599-a413616a293f)

**Comment by timqian** [comment]
这个只能调试一下 prompt, 强调一下必须在代码块中，不过 LLM 不一定总是遵守。我试着强调了下必须在代码块中，似乎可以：


<img width="897" alt="Screenshot 2023-08-19 at 20 33 36" src="https://github.com/timqian/openprompt.co/assets/5512552/e9d12a67-4bc8-4fc6-81b0-c9f559c33b02">


https://openprompt.co/conversations/3192
